app/views/devise/registrations/edit.html.haml

```diff
@@ -1,40 +1,49 @@
 %h2
   Edit #{resource_name.to_s.humanize}
+- if !resource.provider.nil?
```

Comment:
Generally it is preferred to use `unless resource.provider.nil?` instead of `if !`